File: .NET/SystemBenchmarking.Data/Models/Point.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SystemBenchmarking.Data.Models
{
    public class Point
    {
        public double Latitude { get; set; }
        public double Longitude { get; set; }

        public Point()
        {

        }

        public Point(double lat, double lon)
            => (Latitude, Longitude) = (lat, lon);
    }
}

File: monolith/website/distance/point.py
from static.utils import attributes


class Point():
    def __init__(self, latitude: float, longitude: float) -> None:
        self.latitude = latitude
        self.longitude = longitude

    def as_dict(self) -> dict:
        return attributes(self)

    def __str__(self) -> str:
        return f"{self.latitude} {self.longitude}"
